src/js/main.js

```diff
@@ -76,6 +92,7 @@ getData().then((data)=>{
             </div>`;
   }
   divGuestsLoves.innerHTML = homesItems;
```

Comment:
It's not critical, but you can create a new array including first 4 items from `data`. I hope you remember which array method can do this. After that, you can use `map` without `for` loop